lib/src/types.rs

```diff
@@ -120,23 +122,34 @@ pub enum VerificationMethodProperties {
         #[serde(rename = "publicKeyBase58")]
         public_key_base58: String,
     },
-    /// Public key as a Json-Web-Key
-    PublicKeyJwk {
-        #[serde(rename = "publicKeyJwk")]
-        public_key_jwk: String,
-    },
-    /// Public key in Multibase format
-    PublicKeyMultibase {
```

Comment:
did:ethr spec doesn't use these so I removed them from now, since we don't provide a way to convert between byte slices and these formats yet

Reply:
Removing dead code and todo's is always good!